src/comm/TcpAcceptor.cc

```diff
@@ -364,6 +364,9 @@ Comm::TcpAcceptor::oldAccept(Comm::ConnectionPointer &details)
         if (ignoreErrno(errcode)) {
             debugs(50, 5, status() << ": " << xstrerr(errcode));
             return Comm::NOMESSAGE;
+	} else if (ECONNABORTED == errno) {
```

Comment:
Please avoid reusing ephemeral `errno` global if this code stays. Use `errcode` instead. Yes, I know that some of the old code is still dangerously reusing `errno`.